data/data-pipeline/data_pipeline/comparison_tool/src/donut_hole_dacs.yaml

```diff
@@ -0,0 +1,24 @@
+ADDITIONAL_DEMO_COLUMNS:
```

Comment:
Same comment as on the other PR: it doesn't have to be in this PR, but at some point we should add some safety here by loading from YAMLs using a schema like Marshmallow to enforce that the YAML has the right format, acceptable values, etc. Marshmallow is actually pretty easy to work with!

Reply:
added an issue